docs/detections/alerts-view-details.asciidoc

```diff
@@ -65,8 +65,24 @@ If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription
 * *Alerts related by process ancestry* - Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click *Investigate in timeline*.
 
 [discrete]
-[[enriched-data-overview]]
-==== Enriched data on alerts
+[[alerts-enrich-host-user-risk-score]]
+==== Alerts enriched with user and host risk scores
+Alerts can be enriched with user and host risk scores, which convey the level of risk associated with a specific user and host. Risk levels can be `Unknown`, `Low`, `Moderate`, `High`, or `Critical`. Refer to <<host-risk-score>> and <<user-risk-score>> to learn how risk scores are calculated.
+
+IMPORTANT: User and host risk scores are technical preview features. You must enable user and host risk score features to access risk scores data. Refer to [anchor] to learn more.
+
+[role="screenshot"]
+image::images/enriched-host-user-rs.png[Host and user risk score subsections with risk scores, 600]
+
+Two types of user and host risk scores can be shown in the Enriched data section:
+
+* **Current user/host risk classification:** The current risk score of the user or host that is associated with the alert.
+* **Original user/host risk classification:** The first risk score that was calculated for the user or host that is associated with the alert. The *Original user/host risk classification* field always accompanies the *Current user/host risk classification* field to show how the user or host's risk score has changed over time.
```

Comment:
For the original risk score, should we mention that we show it only if it is different form the current?

Reply:
I can be more explicit about this. The behavior is loosely implied in the last sentence so I can modify that.

Reply:
```suggestion
* **Original user/host risk classification:** The first risk score that was calculated for the user or host that is associated with the alert. The *Original user/host risk classification* field only displays if the current risk score no longer matches the original risk score. In this situation, both the current and original risk scores are displayed to show how they changed over time. The current risk score is provided in the *Current user/host risk classification* field. The original risk score is provided in the *Original user/host risk classification* field. 
```